src/components/enterprise-page/EnterprisePage.jsx

```diff
@@ -3,12 +3,24 @@ import PropTypes from 'prop-types';
 import { getAuthenticatedUser } from '@edx/frontend-platform/auth';
 import { AppContext } from '@edx/frontend-platform/react';
 
-const EnterprisePage = ({
-  children,
-  pageContext,
-}) => {
+import { LoadingSpinner } from '../loading-spinner';
+
+import { useEnterpriseCustomerConfig } from './data/hooks';
+
+export default function EnterprisePage({ children }) {
```

Comment:
Not the arrow function! 😢

Reply:
I'm starting to use normal function definitions more now over arrow functions. Unless the function is defined within a component itself, in which case I would use an arrow function. Personal preference I guess 🤷‍♂